includes/privacy-functions.php

```diff
@@ -478,52 +483,53 @@ function edd_register_privacy_exporters( $exporters ) {
 function edd_privacy_customer_record_exporter( $email_address = '', $page = 1 ) {
 
 	$customer    = new EDD_Customer( $email_address );
-	$export_data = array();
-
-	if ( ! empty( $customer->id ) ) {
-		$export_data = array(
-			'group_id'    => 'edd-customer-record',
-			'group_label' => __( 'Customer Record', 'easy-digital-downloads' ),
-			'item_id'     => "edd-customer-record-{$customer->id}",
-			'data'        => array(
-				array(
-					'name'  => __( 'Customer ID', 'easy-digital-downloads' ),
-					'value' => $customer->id
-				),
-				array(
-					'name'  => __( 'Primary Email', 'easy-digital-downloads' ),
-					'value' => $customer->email
-				),
-				array(
-					'name'  => __( 'Name', 'easy-digital-downloads' ),
-					'value' => $customer->name
-				),
-				array(
-					'name'  => __( 'Date Created', 'easy-digital-downloads' ),
-					'value' => $customer->date_created
-				),
-				array(
-					'name'  => __( 'All Email Addresses', 'easy-digital-downloads' ),
-					'value' => implode( ', ', $customer->emails )
-				),
-			)
-		);
 
-		$agree_to_terms_time = $customer->get_meta( 'agree_to_terms_time' );
-		if ( ! empty( $agree_to_terms_time ) ) {
-			$export_data['data'][] = array(
-				'name' => __( 'Agreed to Terms' ),
-				'value' => date_i18n( get_option( 'date_format' ) . ' H:i:s', strtotime( $customer->get_meta( 'agree_to_terms_time' ) ) )
-			);
-		}
+	if ( empty( $customer->id ) ) {
+		return array( 'data' => array(), 'done' => true );
+	}
 
-		$agree_to_privacy_time = $customer->get_meta( 'agree_to_privacy_time' );
-		if ( ! empty( $agree_to_privacy_time ) ) {
-			$export_data['data'][] = array(
-				'name' => __( 'Agreed to Privacy Policy' ),
-				'value' => date_i18n( get_option( 'date_format' ) . ' H:i:s', strtotime( $customer->get_meta( 'agree_to_privacy_time' ) ) )
-			);
-		}
+	$export_data = array(
+		'group_id'    => 'edd-customer-record',
+		'group_label' => __( 'Customer Record', 'easy-digital-downloads' ),
+		'item_id'     => "edd-customer-record-{$customer->id}",
+		'data'        => array(
+			array(
+				'name'  => __( 'Customer ID', 'easy-digital-downloads' ),
+				'value' => $customer->id
+			),
+			array(
+				'name'  => __( 'Primary Email', 'easy-digital-downloads' ),
+				'value' => $customer->email
+			),
+			array(
+				'name'  => __( 'Name', 'easy-digital-downloads' ),
+				'value' => $customer->name
+			),
+			array(
+				'name'  => __( 'Date Created', 'easy-digital-downloads' ),
+				'value' => $customer->date_created
+			),
+			array(
+				'name'  => __( 'All Email Addresses', 'easy-digital-downloads' ),
+				'value' => implode( ', ', $customer->emails )
+			),
+		)
+	);
+
+	$agree_to_terms_time = $customer->get_meta( 'agree_to_terms_time' );
+	if ( ! empty( $agree_to_terms_time ) ) {
+		$export_data['data'][] = array(
+			'name' => __( 'Agreed to Terms' ),
```

Comment:
Looks like this is already in `master`, but the text domain is missing.

Reply:
@SDavisMedia yeah, this was part of that merging issue branches before merging master thing. I'll get those fixed up.